modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc

```diff
@@ -196,9 +196,9 @@ You can reference the ARN of your KMS key when you create the cluster in the nex
 
 . Create a cluster with STS using custom installation options. You can use the `--interactive` mode to interactively specify custom settings:
 +
-[IMPORTANT]
+[WARNING]
 ====
-You cannot install a ROSA cluster into an existing managed VPC. Managed VPCs are created during the managed cluster deployment process, and must only be associated with a single cluster to ensure that cluster provisioning and deletion operations work correctly. To determine whether a VPC is managed, look for the `red-hat-managed` tag; managed VPCs are tagged with `red-hat-managed:true`.
+You cannot install a ROSA cluster into an existing VPC that was created by the OpenShift installer. These VPCs are created during the cluster deployment process, and must only be associated with a single cluster to ensure that cluster provisioning and deletion operations work correctly. To determine whether a VPC was created by the OpenShift installer, check for the "owned" value on the `kubernetes.io/cluster/<infra-id>` tag, for example, when viewing the tags for the VPC called `mycluster-12abc-34def`, the `kubernetes.io/cluster/mycluster-12abc-34def` tag has a value of `owned`, so it was created by the installer and must not be modified by the administrator.
```

Comment:
A few suggestions intended to improve clarity (applies to all instances of this warning block):

```suggestion
You cannot install a ROSA cluster into an existing VPC that was created by the OpenShift installer. These VPCs are created during the cluster deployment process and must only be associated with a single cluster to ensure that cluster provisioning and deletion operations work correctly. To verify whether a VPC was created by the OpenShift installer, check for the `owned` value on the `kubernetes.io/cluster/<infra-id>` tag. For example, when viewing the tags for the VPC named `mycluster-12abc-34def`, the `kubernetes.io/cluster/mycluster-12abc-34def` tag has a value of `owned`. Therefore, the VPC was created by the installer and must not be modified by the administrator.
```

I would also consider splitting this into two paragraphs for visual clarity, with "To verify..." starting the second paragraph.

Reply:
Much appreciated, thanks Pan!